readme.md

```diff
@@ -18,6 +18,19 @@ Using jQuery, write code that makes an AJAX get request to "http://kittengifs.co
 Your Answer:
 ```js
 
+$(document).ready(function(){
+  // ajax get
+  $(".test_ajax_get").on("click", function(){
```

Comment:
No need for a click listener, that's outside the scope of this question. 

Otherwise, this looks correct to me.